infrastructure/code/code_html.go

```diff
@@ -46,27 +47,29 @@ func getDataFlowHeadingHtml(issue snyk.CodeIssueData) string {
 }
 
 func getDataFlowHtml(issue snyk.CodeIssueData) string {
-	dataFlowHtml := ""
+	dataFlowHtml := `<table class="data-flow-body"><tbody>`
+
 	for i, flow := range issue.DataFlow {
 		fileName := filepath.Base(flow.FilePath)
 		dataFlowHtml += fmt.Sprintf(`
-		<div class="data-flow-row">
-		  <span class="data-flow-number">%d</span>
-		  <span class="data-flow-blank"> </span>
-		  <span class="data-flow-filepath" file-path="%s" start-line="%d" end-line="%d" start-character="%d" end-character="%d">%s:%d</span>
-		  <span class="data-flow-delimiter">|</span>
-		  <span class="data-flow-text">%s</span>
-		</div>`,
+		  <tr class="data-flow-row">
+		    <td class="data-flow-number">%d</td>
+		    <td class="data-flow-clickable-row" file-path="%s" start-line="%d" end-line="%d" start-character="%d" end-character="%d">%s:%d</td>
```

Comment:
If you change the class here we will need to change the logic in https://github.com/snyk/snyk-intellij-plugin/pull/497 a bit so we're getting the right class. Just FYI, depends who merges their PR first 😆

Reply:
I'll do it in https://github.com/snyk/snyk-intellij-plugin/pull/497